ledger/participant-state/kvutils/app/src/test/scala/com/daml/ledger/participant/state/kvutils/app/RunnerSpec.scala

```diff
@@ -160,12 +160,16 @@ object RunnerSpec {
     ParticipantConfig(
       mode = ParticipantRunMode.Combined,
       participantId = participantId,
-      shardName = None,
+      shardName = Some(UUID.randomUUID().toString),
```

Comment:
Out of interest, what is this needed for?

Reply:
so in the previous implementation createMetrics call was overriden to always create new metric registry in `TestConfigProvider` not to clash for Jvm metrics while here i use production version of createMetrics call which supports shards, so I generate random shardName and metrics do not clash.